tests/verifier.rs

```diff
@@ -374,10 +374,27 @@ fn test_verifier_err_unknown_opcode() {
 }
 
 #[test]
-#[should_panic(expected = "InvalidFunction(1811268607)")]
+#[should_panic(expected = "UnknownOpCode(6, 0)")]
```

Comment:
Are we testing the invalid exit? Shouldn't we be throwing an error for 0x95?

Reply:
`test_verifier_err_invalid_return` appears to have the same issue.

Reply:
You are right, will rewrite these two tests.